### ISSUE TITLE
Ritual spells are given blue borders

### ISSUE THREAD

**Issue by monyarm** [piface314/ygo-fabrica#3]
```

[card.ongeki-henshin]
id = 19710004
name = "Ongeki Henshin"
type = "spell ritual" 
effect = '''
This card can be used to Ritual Summon any "Kamen Rider" Ritual Monster. You must also Tribute monsters from your hand or field, whose total Levels equal or are greater than the Level of the "Kamen Rider" Ritual Monster you Ritual Summon.'''
```

ygopic/composer turns the above in to a ritual spell card with a blue border.

**Comment by piface314** [comment]
Thank you for pointing that out! I hadn't noticed this bug before, and it will be fixed in the next release. ;)

**Comment by piface314** [comment]
This issue has been fixed in bbd58a3e65d281f492ee95b1726dae59c7929d04, and will be closed as soon as v2.0 is released.
